misc/dash_api_utils

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import click
+import sys
+
+import dash_api.utils as utils
+
+
+@click.command()
+@click.option("--to_json", "transformation", flag_value="json", default=True,
+              show_default=True, help="Convert input content to json.")
+@click.option("--to_pb", "transformation", flag_value="pb",
```

Comment:
>to_pb [](http://example.com/codeflow?start=17&length=5)

since this is long option, call it `to_protobuf` or `to_proto`?

Reply:
Done